src/frontends/pytorch/src/op/unique.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/unique.hpp"
+
+#include <memory>
+
+#include "openvino/core/node.hpp"
+#include "openvino/core/node_output.hpp"
+#include "openvino/core/node_vector.hpp"
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/gather.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/slice.hpp"
+#include "openvino/op/subtract.hpp"
+#include "openvino/op/unique.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+using namespace ov::op;
+
+Output<Node> flip(const NodeContext& context, const Output<Node>& x, const Output<Node>& axis) {
+    auto minus_one = context.mark_node(v0::Constant::create(element::i32, Shape{1}, {-1}));
+    auto minimum_int =
+        context.mark_node(v0::Constant::create(element::i32, Shape{1}, {std::numeric_limits<int>::min()}));
+    auto slice = context.mark_node(std::make_shared<v8::Slice>(x, minus_one, minimum_int, minus_one, axis));
+    return slice;
+};
```

Comment:
```suggestion
namespace {
Output<Node> flip(const NodeContext& context, const Output<Node>& x, const Output<Node>& axis) {
    auto minus_one = context.mark_node(v0::Constant::create(element::i32, Shape{1}, {-1}));
    auto minimum_int =
        context.mark_node(v0::Constant::create(element::i32, Shape{1}, {std::numeric_limits<int>::min()}));
    auto slice = context.mark_node(std::make_shared<v8::Slice>(x, minus_one, minimum_int, minus_one, axis));
    return slice;
};
}  // namespace
```
Please wrap helper functions in nameless namespace